cockatrice/src/player.cpp

```diff
@@ -682,9 +682,11 @@ void Player::retranslateUi()
     aPlay->setText(tr("&Play"));
     aHide->setText(tr("&Hide"));
     aPlayFacedown->setText(tr("Play &Face Down"));
+    //: Turn sideways or back again
     aTap->setText(tr("&Tap / Untap"));
     aDoesntUntap->setText(tr("Toggle &normal untapping"));
-    aFlip->setText(tr("&Flip"));
+    //: Turn face up/face down
+    aTurnOver->setText(tr("&Turn Over"));
```

Comment:
This won't work; We have 2 &T's now. I honestly dislike the windows automated shortcuts as they cause translations to be a PITA

Reply:
Good catch! Will change that.

Is it windows only? I think it's a general way to access menus on all platforms?

But wait... does this work for right click menus at all?
For the main menus at top sure, no problem. But here I didn't get it to work in a local test game.

Any ideas? Could we remove it from all right click menus?